docs/deployment.md

```diff
@@ -0,0 +1,93 @@
+means you can run functions on Cloud Run with ease.
+
+There are detailed instructions for C# available as part of the
+Cloud Run documentation:
+
+- [Quickstart for C#](https://cloud.google.com/run/docs/quickstarts/build-and-deploy#c)
+- [Codelab for C#](https://codelabs.developers.google.com/codelabs/cloud-run-hello-csharp/index.html)
+
+The instructions apply to functions in exactly the same way as
+normal, with the same Dockerfile configuration. The `PORT`
+environment variable will be supplied by Cloud Run. If your project
+contains multiple functions, the simplest approach is to specify an
+environment variable in your Dockerfile, with a line like this:
+
+```text
+ENV FUNCTION_TARGET HelloFunctions.Function
+```
+
+## Deploying to Google Cloud Functions
+
+> **NOTE:**  
+> The `dotnet3` runtime is currently available on an "allow
+> list" basis. Deployment will fail unless your project has been
+> specifically included in the list.
+
+Deployment to Google Cloud Functions is easy with the `gcloud`
+command line. See the [general deployment
+documentation](https://cloud.google.com/functions/docs/deploying/filesystem)
+for more detailed information; this page is only intended to give simple
+"getting started" instructions. In general, deploying a .NET
+Functions Framework function follows the same procedure as deploying
+a function written in every language. The .NET-specific aspects are:
+
+- The runtime should be specified as `dotnet3`
+- The entry point should be specified as the name of the function
+  type, including namespace (e.g. `HelloFunctions.Function`)
+
+The command line options are used to specify how the function is
+triggered.
+
+### HTTP functions
+
+HTTP functions are deployed using `--trigger-http`. For example:
+
+```text
+gcloud functions deploy hello-functions --runtime dotnet3 --trigger-http --entry-point HelloFunctions.Function
+```
+
+On successful deployment, details of the deployed function will be
+displayed, including the URL. Visit the URL in a browser to invoke
+your function.
+
+### Legacy event functions
+
+When you deploy a function listening for a particular event, you
+have to specify the *event trigger* as part of the deployment
+(instead of `--trigger-http` as above).
+
+**Sample triggers**
+
+Replace any ***bold italic*** parts with your project ID, bucket name and so forth.
+
+Trigger type             | Payload type   | Sample command line options
+------------------------ | -------------- | --------------------------
+Cloud Storage operation  | StorageObject  | --trigger-event google.storage.object.finalize --trigger-resource ***my-gcs-bucket***
+Pub/Sub message          | PubSubMessage  | --trigger-topic ***my-pubsub-topic-id***
+Firestore event          | FirestoreEvent | --trigger event providers/cloud.firestore/eventTypes/document.write --trigger-resource 'projects/***my-project***/databases/(default)/documents/***my-collection***/{document}'
```

Comment:
Is there a dash missing? `--trigger event `.

Reply:
Well spotted - fixed, thanks.